internal/controllers/clusterextension_controller_test.go

```diff
@@ -122,12 +135,10 @@ func TestClusterExtensionNonExistentVersion(t *testing.T) {
 	cond = apimeta.FindStatusCondition(clusterExtension.Status.Conditions, ocv1alpha1.TypeInstalled)
 	require.NotNil(t, cond)
 	require.Equal(t, metav1.ConditionUnknown, cond.Status)
-	require.Equal(t, ocv1alpha1.ReasonInstallationStatusUnknown, cond.Reason)
-	require.Equal(t, "installation has not been attempted due to failure to gather data for resolution", cond.Message)
+	//require.Equal(t, "installation has not been attempted due to failure to gather data for resolution", cond.Message) // ??
```

Comment:
this seemed very specific and not set by a corresponding set in the codebase elsewhere, so I think we can live without it in the test.

Reply:
If I'm guessing right, I think this was to capture error while building dippy variables for resolution. We can remove this.

Reply:
changed in c6d6f078